### ISSUE TITLE
[Bug] Can't watch Youtube videos

### ISSUE THREAD

**Issue by Nikodemsky** [SpikeHD/Dorion#208]
**Describe the bug**
Any video in any channel doesn't start - it's just trying to load infinitely

**To Reproduce**
Just playing any video at all.

**Expected behavior**
It should work?

**Screenshots**
![YT](https://github.com/SpikeHD/Dorion/assets/14090596/33fc11df-994c-4b4c-9e23-8d7eb4d6fefd)

**System Info (please complete the following information):**
 - OS: Win10 Pro 22H2
 - Version: 4.1.2

**Additional context**
Never had this problem on the native client or Armcord (checked on native client today), just Dorion for some reason doesn't want to play any youtube videos. Using Dorion since 4.1.1 and always had this problem.

Not using any kind of VPN or anything like that.

**Comment by yavko** [comment]
same

**Comment by KonerDev** [comment]
I also have the same problem. The dev console shows this error:
```
Uncaught DOMException: Failed to construct 'PresentationRequest': The document is sandboxed and lacks the 'allow-presentation' flag.
    at W.initialize (https://www.gstatic.com/eureka/clank/118/cast_sender.js:83:87)
    at chrome.cast.initialize (https://www.gstatic.com/eureka/clank/118/cast_sender.js:95:137)
    at g.k.init (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:569:13)
    at PDb (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:252:40)
    at FDb (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:258:9)
    at IDb.d.disableCastApi.window.chrome.chrome.cast.chrome.cast.isAvailable.window.__onGCastApiAvailable (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:246:402)
    at chrome.cast.da (https://www.gstatic.com/eureka/clank/118/cast_sender.js:97:297)
```

To get around this problem, I currently just click the YouTube button at the bottom right of the video player to open the video in a popup.

**Comment by yavko** [comment]
> I also have the same problem. The dev console only shows this error (It probably has nothing to do with the error as it can also be seen on the Discord website):
> 
> ```
> Uncaught DOMException: Failed to construct 'PresentationRequest': The document is sandboxed and lacks the 'allow-presentation' flag.
>     at W.initialize (https://www.gstatic.com/eureka/clank/118/cast_sender.js:83:87)
>     at chrome.cast.initialize (https://www.gstatic.com/eureka/clank/118/cast_sender.js:95:137)
>     at g.k.init (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:569:13)
>     at PDb (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:252:40)
>     at FDb (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:258:9)
>     at IDb.d.disableCastApi.window.chrome.chrome.cast.chrome.cast.isAvailable.window.__onGCastApiAvailable (https://www.youtube.com/s/player/e04b3b4c/player_ias.vflset/de_DE/remote.js:246:402)
>     at chrome.cast.da (https://www.gstatic.com/eureka/clank/118/cast_sender.js:97:297)
> ```
> 
> To get around this problem, I currently just click the YouTube button at the bottom right of the video player to open the video in a popup.

this error is not the cause of the youtube video not playing, i did some testing to prove so
![image](https://github.com/SpikeHD/Dorion/assets/15178513/ed471073-3fa4-4587-b2cd-116735536adf)
> no error occurs when adding the sandbox flag

**Comment by SpikeHD** [comment]
Fixed as of 8c9b2f24c35354435ebe8ae4d1f7661c36d4085a, will be available in the next release 👍
